app/assets/stylesheets/profiles.scss

```diff
@@ -0,0 +1,12 @@
+.user_avatar {
+  display: inline-block;
+}
+.file.optional.user_avatar {
+  margin: 0.5em 0;
+}
+a.navbar-link figure.image {

```

Comment:
for example `a.navbar-link figure.image {` .... 
this style will affect ALL places where nav-link is used, in header, footer, everywhere .... but if you need only on this page - put in some parent class or #container